utils/functions.py

```diff
@@ -449,7 +449,7 @@ async def update_known_user(the_user):
                 "$set": {
                     "username": the_user.name,
                     "discriminator": the_user.discriminator,
-                    "avatar": the_user.display_avatar.url,
+                    "avatar": the_user.display_avatar.key,
```

Comment:
If the user has the default avatar this will save an int `[0..4]` rather than a valid avatar hash (it should save None - maybe use a check on [`the_user.avatar`](https://docs.disnake.dev/en/latest/api.html#disnake.User.avatar))